### DISCUSSION TITLE
How to pass arbitrary values to "content"?

### DISCUSSION THREAD

**Discussion by zFlxw** [tailwindlabs/tailwindcss#10847] [Help] [answered]
Hey,

Let's assume following code:
`<p class={"before:content-['§{somestuff}']"}></p>`
(the string literals are replaced so that the codeblock does render correctly)

This worked for me at first, but after some time it randomly broke and stopped working.
After doing some research I found out, that tailwind does only support full class names and that you cannot insert an arbitrary value in middle of a class name.

Now is my question, how can I get this working the best way?

**Comment by wongjn** [comment, marked as answer]
You could use a CSS variable and the `style` attribute, like:
```html
<p class="before:content-[var(--content)]" style={"--content: '§{somestuff}'"}>
```

**Reply by zFlxw** [reply]
Thank you very much, this works!
